nicoRDL/dotTheory-JS
# JS-301 (JS-354)  AJAX Basics: Programming AJAX

## Related Ticket

https://codeandtheory.atlassian.net/browse/JS-354

## Description

- completed all sample problems and code challenge for programming callback functions and handling an AJAX response
- completed all AJAX quizzes

## Steps to Validate
1. Open the codeChallenge folder, open the index.html file
2. On the sidebar, a Meeting Rooms section should be visible with Rooms and their corresponding availability color
3. Open the roomWidget.js file; evaluate
4. The AJAX request should be a valid request

<img width="1280" alt="Screenshot 2023-07-06 at 4 52 29 PM" src="https://github.com/nicoRDL/dotTheory-JS/assets/100089185/7821759c-b1a0-42e6-848d-cb41f236f57b">

## Risks
N/A
